report/id.go

```diff
@@ -175,6 +175,15 @@ func ParsePodNodeID(podNodeID string) (namespaceID, podID string, ok bool) {
 	return fields[0], fields[1], true
 }
 
+// ParseServiceNodeID produces the namespace ID and service ID from an service node ID.
+func ParseServiceNodeID(serviceNodeID string) (namespaceID, serviceID string, ok bool) {
+	fields := strings.SplitN(serviceNodeID, ScopeDelim, 2)
+	if len(fields) != 2 {
+		return "", "", false
+	}
+	return fields[0], fields[1], true
+}
+
```

Comment:
This is probably just in for consistency, but it's not actually used anywhere...